js/hang-ui/src/Components/publish/CameraSourceButton.tsx

```diff
@@ -8,20 +8,23 @@ export default function CameraSourceButton() {
 		const hangPublishEl = context?.hangPublish();
 		if (!hangPublishEl) return;
 
-		if (hangPublishEl.source === "camera") {
+		if (hangPublishEl.source.peek() === "camera") {
 			// Camera already selected, toggle video.
-			hangPublishEl.video = !hangPublishEl.video;
+			hangPublishEl.invisible.set(!hangPublishEl.invisible.peek());
 		} else {
-			hangPublishEl.source = "camera";
-			hangPublishEl.video = true;
+			hangPublishEl.source.set("camera");
+			hangPublishEl.invisible.set(false);
```

Comment:
I decided to flip the default around, so audio/video are enabled by default and you need to explicitly disable them.